packages/mc-scripts/src/commands/build.ts

```diff
@@ -0,0 +1,153 @@
+import path from 'path';
+import fs from 'fs-extra';
+import webpack, { type Configuration, type Stats } from 'webpack';
+import chalk from 'chalk';
+import checkRequiredFiles from 'react-dev-utils/checkRequiredFiles';
```

Comment:
nit: I noticed that the type declaration file was not recognized in IDE until I added to `tsconfig.json`:
```json
{
  "extends": "@tsconfig/node16/tsconfig.json",
  "compilerOptions": {
    "typeRoots": ["../../@types", "../../@types-extensions"],
  }
}
```